packages/divination-oracles/0.1.0/package.yml

```diff
@@ -0,0 +1,380 @@
+# Returns a random playing card from a standard 52 card deck, with no jokers
+matches:
+  - trigger: ":card:"
+    replace: "{{output}}"
+    vars:
+      - name: output
+        type: random
+        params:
+          choices:
+            - "Ace of Hearts"
```

Comment:
Any reason why this one isn't constructed of _two_ shorter random choice params combining Suit and Value? 😃

Reply:
Good point! It must have not occurred to me